### DISCUSSION TITLE
PR3 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#375] [General]
Reviewer: @tdegeorge 

1 - show work for long division
2-5 enter answers labeled as quotient and remainder in two boxes
6-8 enter answers as X + Y/divisor in two boxes
9-11 enter answers as X+Y/divisor in one box

I don't think we need all of these, maybe the instructor can choose which type they like for entering answers from 2-5, 6-8, or 9-11.

**Comment by tdegeorge** [comment]
Looks good to me! I think it would be good to keep them all and then the instructor can just pick and choose.
